### PR TITLE
Frontend doc link

### PR DESCRIPTION
**Pull request by saif-ericsson**
Adding links to frontend subscription documentation

Issue #398 

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: wiki/markdown/subscription-API.md [hunk 1]
```diff
@@ -97,3 +97,12 @@ Delete a single specific subscription
 Curl command example
 
     curl -X DELETE -H "Content-type: application/json"  http://<host>:8090/subscriptions/<name>
+    
+   
+ ## Subscription related information in front-end documentation
+ 
+Read more about curl examples for subscription endpoint in Eiffel Intelligence Frontend docs [here]
```
Comment (e-pettersson-ericsson) [inline wiki/markdown/subscription-API.md]: Frontend --> front-end 😄 and there shouldn't be spaces between the [] and ()

File: src/test/java/com/ericsson/ei/jmespath/test/TestStringSplittingFunction.java [hunk 2]
```diff
@@ -37,7 +37,10 @@ public void testStringSplit() throws Exception {
 
     @Test
     public void testStringSplitFunction() throws Exception {
-        String processRule = "split(data.identity, '/')";
+        // String processRule = "split(data.identity, '/')";
+
+        String processRule = "split(data.identity, '@|/')";
```
Comment (a-breid-ericsson) [inline src/test/java/com/ericsson/ei/jmespath/test/TestStringSplittingFunction.java]: Why this change?

File: src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java [hunk 3]
```diff
@@ -83,6 +83,7 @@
 
     private ObjectMapper objMapper;
 
+
```
Comment (a-breid-ericsson) [inline src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java]: Why an extra line here?

File: wiki/markdown/subscription-API.md [hunk 4]
```diff
@@ -97,3 +97,11 @@ Delete a single specific subscription
 Curl command example
 
     curl -X DELETE -H "Content-type: application/json"  http://<host>:8090/subscriptions/<name>
+    
+   
+ ## Subscription related information in front-end documentation
+ 
+Read more about curl examples for subscription endpoint in Eiffel Intelligence front-end docs [here](https://github.com/eiffel-community/eiffel-intelligence-frontend/blob/master/wiki/markdown/curl-examples.md#subscriptions).
+
```
Comment (a-breid-ericsson) [inline wiki/markdown/subscription-API.md]: Please add a title for this link as well since previous title states that it links to fron and but this link does not.

File: src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java (hunk 3 above)
Comment (saif-ericsson) [reply src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java]: removed in next commit

File: src/test/java/com/ericsson/ei/jmespath/test/TestStringSplittingFunction.java (hunk 2 above)
Comment (saif-ericsson) [reply src/test/java/com/ericsson/ei/jmespath/test/TestStringSplittingFunction.java]: it was unintended merge.... fixed in next commit

Comment (saif-ericsson) [reply src/test/java/com/ericsson/ei/jmespath/test/TestStringSplittingFunction.java]: I removed it as it was not required.

File: wiki/markdown/subscription-API.md (hunk 4 above)
Comment (saif-ericsson) [reply wiki/markdown/subscription-API.md]: I removed it as it is not required. next commit
